docs/src/main/sphinx/connector/jmx.rst

```diff
@@ -32,17 +32,17 @@ To enable periodical dumps, define the following properties:
     jmx.max-entries=86400
 
 ``dump-tables`` is a comma separated list of Managed Beans (MBean). It specifies
-which MBeans is sampled and stored in memory every ``dump-period``.
-History has limited size of ``max-entries`` of entries. Both ``dump-period``
-and ``max-entries`` have default values of ``10s`` and ``86400`` accordingly.
+which MBeans are sampled and stored in memory every ``dump-period``. History
+has a limited size of ``max-entries`` number of entries. Both ``dump-period``
+and ``max-entries`` have default values of ``10s`` and ``86400`` respectively.
 
-Commas in MBean names should be escaped in the following manner:
+Commas in MBean names should be escaped using backslashes (``\``) in the
```

Comment:
names must be escaped

Reply:
here you have one backslash and then you have \\ below
maybe explain that backslaches need to be \-escaped in .properties file?